lxd/container.go

```diff
@@ -729,6 +729,8 @@ type container interface {
 	CurrentIdmap() (*idmap.IdmapSet, error)
 	DiskIdmap() (*idmap.IdmapSet, error)
 	NextIdmap() (*idmap.IdmapSet, error)
+
+	InsertSeccompUnixDevice(prefix string, m types.Device, pid int) error
```

Comment:
Should go under the FIXME header above as this isn't a function we'll want to keep when we switch to the instance interface, so we'll need to change it to something else then.

Reply:
I meant move it up to be after DaemonState, no need to keep it at the end of the interface definition